app/src/index.js

```diff
@@ -2,26 +2,27 @@ import React from "react";
 import ReactDOM from "react-dom";
 
 import { Auth0Provider } from "@auth0/auth0-react";
+
 import "./styles.css";
 
-import App from "./App";
+import App, { history } from "./App";
```

Comment:
Totally optional, but fun fact! This would also work `import { App, history } from './App';` if you remove the `default` label from `App.js`
https://github.com/LeiaQuesada/MyGarden/blob/0484e3052478434e2f516591f223f40a07ed3b25/app/src/App.js#L18

It's like the same thing going on in `db.mjs`, where you _could_ import just a few, or import them all like how we do with `import * as db from 'db.mjs'`